algoliasearch-common/src/main/java/com/algolia/search/APIClient.java

```diff
@@ -807,6 +807,12 @@ private TaskSingleIndex batchSingleIndex(
 
   Task deleteObject(String indexName, String objectID, RequestOptions requestOptions)
       throws AlgoliaException {
+
+    if (objectID.trim().length() == 0)
+    {
```

Comment:
😱 
We are using google formatted for java. You can run `mvn com.coveo:fmt-maven-plugin:format` to format the code automatically

Reply:
Yup! Will fix it.